ynput/ayon-core
# USD Contribution Workflow: Enable asset contributions to write AYON Entity URIs

## Changelog Description

Enable the writing of Ayon Entity URIs for asset/shot contribution workflows for the AYON USD resolver to resolve.

## Additional info

Must be tested using the [AYON USD Resolver](https://github.com/ynput/ayon-usd-resolver)

### TODO

- [x] Implement toggle in settings to enable/disable.

## Testing notes:

1. Enable the `use_ayon_entity_uri` Entity URI in settings.

```
ayon+settings://core/publish/ExtractUSDAssetContribution/use_ayon_entity_uri
ayon+settings://core/publish/ExtractUSDLayerContribution/use_ayon_entity_uri
```

3. Publish USD files with asset or shot contributions (e.g. from Houdini, but should also work from Maya and Blender!)
